language/tools/vm-genesis/src/genesis_context.rs

```diff
@@ -5,125 +5,10 @@
 
 use anyhow::Result;
 use libra_state_view::StateView;
-use libra_types::{
```

Comment:
@runtian-zhou @tzakian not sure who added this but let me flag it just in case :)